alvr/session/src/settings.rs

```diff
@@ -159,27 +159,13 @@ Temporal: Helps improve overall encoding quality, very small trade-off in speed.
     pub enable_weighted_prediction: bool,
 }
 
+#[derive(SettingsSchema, Serialize, Deserialize, Clone, PartialEq)]
+#[schema(collapsible)]
+pub struct AmdConfig {}
+
```

Comment:
Huh? This doesn't appear to be used anywhere

Reply:
Because it's pub. Sometimes rustc/clippy catches it but anyway but i don't know the rule (maybe when it's not seen outside the crate?)